packages/truffle/cli.webpack.config.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
   devtool: 'source-map',
   module: {
     rules: [
-      { test: /\.js$/, use: "shebang-loader" }
+      { test: /\.js$/, use: "shebang-loader" },
+      { test: /rx\.lite\.aggregates\.js/, use: 'imports-loader?define=>false' }
```

Comment:
Where do these files come from? Generated by sinon?

Reply:
(Just wondering about the context for this + the imports-loader thing)

Reply:
turns out some of the checks were failing when I added `inquirer` because `rx-lite-aggregates` (part of the `rxjs` dep for `inquirer`) was calling for `'./rx.lite` instead of `rx-lite`.